scripts/upgrade

```diff
@@ -182,7 +182,7 @@ ynh_multimedia_addaccess $app
 ynh_script_progression --message="Upgrading logrotate configuration..." --weight=1
 
 # Use logrotate to manage app-specific logfile(s)
-ynh_use_logrotate
+ynh_use_logrotate --non-append
```

Comment:
Apparently this option is deprecated. It was failing at some point. see apps chatroom log for more detail